### PR TITLE
RH-SSO client role support

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

**Review by tomasdavidorg — APPROVED** [review]

File: doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc [hunk 1]
```diff
@@ -70,6 +77,13 @@ In this example:
 The RH-SSO server converts the user names to lower case. Therefore, after integration with RH-SSO, your user name will appear in lower case in {PRODUCT}. If you have user names in upper case hard coded in business processes, the application might not be able to identify the upper case user.
 ====
 +
+If you want to use client roles, also add the following setting udner `<secure-deployment>`:
```
Comment (RishiRajAnand) [inline doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc:80]: "under"

**Review by RishiRajAnand — CHANGES_REQUESTED** [review]

**Comment by RishiRajAnand** [comment]
Looks good to me.

File: doc-content/enterprise-only/integration/sso-user-add-proc.adoc [hunk 2]
```diff
@@ -22,4 +22,6 @@ For projects that deploy from {CENTRAL} on OpenShift, create an RH-SSO user call
 +
 . Define the roles as realm roles in the *Realm Roles* tab under the *Roles* section.
 +
+Alternatively, you can define the roles as client roles for the `kie` client. For instructions about configuring the `kie` client, see <<sso-central-client-create-proc>>. To use client roles, you must also configure additional settings for {CENTRAL}, as described in <<sso-client-adapter-proc>>.
```
Comment (sterobin) [inline doc-content/enterprise-only/integration/sso-user-add-proc.adoc:25]: At some point you'll want to not use this legacy `<<anchor-id>>` format and instead use the prescribed `xref:anchor-id[]` and/or `xref:anchor-id_context-id[]` format. That legacy format has issues with context IDs, which in theory need to be in every module anyway (which I see isn't the case here yet). Just to keep on your radar.

**Review by sterobin — APPROVED** [review]
@mramendi Done, one thing.

File: doc-content/enterprise-only/integration/sso-user-add-proc.adoc (hunk 2 above)
Comment (mramendi) [reply doc-content/enterprise-only/integration/sso-user-add-proc.adoc:25]: Thanks, added to https://issues.redhat.com/browse/BXMSDOC-7163 - I see no reason to change this in just one place,I need to do it in the entire doc as a part of its improvement (and, very likely, single-sourcing).

**Pull request merged**